### ISSUE TITLE
TypeError: 'bool' object is not callable with "panaroo-merge"

### ISSUE THREAD

**Issue by alexmsalmeida** [gtonkinhill/panaroo#74]
Hi,

I am trying to use the program panaroo-merge to combine multiple independent analyses but I seem to be getting an error with various different combinations of directories.

Whenever it reaches the "Generating output..." step I get the following error:

```
Generating output...
Traceback (most recent call last):
  File "/nfs/production/interpro/metagenomics/mags-scripts/miniconda3/envs/func-annot/bin/panaroo-merge", line 10, in <module>
    sys.exit(main())
  File "/nfs/production/interpro/metagenomics/mags-scripts/miniconda3/envs/func-annot/lib/python3.7/site-packages/panaroo/merge_graphs.py", line 570, in main
    quiet=args.quiet)
  File "/nfs/production/interpro/metagenomics/mags-scripts/miniconda3/envs/func-annot/lib/python3.7/site-packages/panaroo/merge_graphs.py", line 328, in merge_graphs
    G = merge_paralogs(G)
TypeError: 'bool' object is not callable
```

Seems to be related with the merge_paralogs function. This is the command I used to run each analysis individually:

```
panaroo -t 32 -i genome_list.txt -o analysisX --clean-mode strict --merge_paralogs --core_threshold 0.90 -c 0.90 -f 0.5
```

Then this is the one I used to merge the analyses:

```
panaroo-merge -t 32 -d analysis1 analysis2 -o analysis_combined --merge_paralogs --core_threshold 0.90 -c 0.90 -f 0.5
```

Am I doing something wrong? To clarify I am using panaroo 1.2.3.

Thanks in advance for the help.

Alex

**Comment by gtonkinhill** [comment]
Hi Alex,

I suspect this might be a bug caused by combining the graph merge with merging paralogs. I have not tested the merge script as thoroughly so there is still the occasional issue that needs fixing.

I am planning on creating a release this week to deal with a couple of minor issues so will attempt to include a fix for this then.

I would also recommend running panaroo with the `--no_clean_edges` option enabled if you are later planning to run `panaroo-merge`. This is because trimming rare edges as is default in panaroo may reduce the effectiveness of the merge step.

**Comment by alexmsalmeida** [comment]
Great, thanks for the tip. I'll keep an eye out for the new release.

Thanks,
Alex

**Comment by gtonkinhill** [comment]
Thanks for waiting. This should be fixed in v1.2.4
